crates/bitwarden-exporters/src/cxf/login.rs

```diff
@@ -50,8 +46,45 @@ pub(super) fn to_login(
                 creation_date,
             }]
         }),
-    };
-    login
+    }
+}
+
+/// Converts a `CredentialScope` to a vector of `LoginUri` objects.
+///
+/// This is used for login credentials.
+fn to_uris(scope: &CredentialScope) -> Vec<LoginUri> {
+    let urls = scope.urls.iter().map(|u| LoginUri {
+        uri: Some(u.clone()),
+        r#match: None,
+    });
+
+    let android_apps = scope.android_apps.iter().map(|a| LoginUri {
+        uri: Some(format!("{}://{}", ANDROID_APP_SCHEME, a.bundle_id)),
```

Comment:
This is will throw a clippy lint once we update to rust 1.88
```suggestion
        uri: Some(format!("{ANDROID_APP_SCHEME}://{}", a.bundle_id)),
```